code/sound/speech.mm

```diff
@@ -0,0 +1,171 @@
+#ifdef FS2_SPEECH
+
+#import <AppKit/AppKit.h>
+#import <AppKit/NSSpeechSynthesizer.h>
+
+#include "globalincs/pstypes.h"
+#include "utils/unicode.h"
+
+
+static NSSpeechSynthesizer *synth = nil;
+static bool Speech_init = false;
+
+
+bool speech_init()
+{
+	if (Speech_init) {
+		return true;
+	}
+
+	synth = [[NSSpeechSynthesizer alloc] init];
+
+	Speech_init = true;
+
+	return true;
+}
```

Comment:
Looks like this function can only return true.  Is the return value necessary?

Reply:
It has to match the windows implementation, which returns a lot of bool's for some reason.

Reply:
As suspected.